weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx

```diff
@@ -244,67 +249,101 @@ export const SimplePageLayoutWithHeader: FC<{
         {simplePageLayoutContextValue.headerSuffix}
       </Box>
       <div style={{flex: '1 1 auto', overflow: 'hidden'}}>
-        <SplitPanel
+        <TraceTreeSplitPanel
           minWidth={150}
           defaultWidth={200}
           maxWidth="50%"
-          isDrawerOpen={props.isSidebarOpen ?? false}
-          drawer={props.leftSidebar}
+          isDrawerOpen={props.isLeftSidebarOpen ?? false}
+          drawer={props.leftSidebarContent}
           main={
-            <Box
-              sx={{
-                display: 'flex',
-                flexDirection: 'column',
-                flexGrow: 1,
-                height: '100%',
-                overflow: 'hidden',
-              }}>
-              <Box
-                sx={{
-                  maxHeight: '50%',
-                  flex: '0 0 auto',
-                  width: '100%',
-                  overflow: 'auto',
-                  pt: 1,
-                  px: 2,
-                  alignContent: 'center',
-                }}>
-                {props.headerContent}
-              </Box>
-              {(!props.hideTabsIfSingle || tabs.length > 1) && (
-                <Tabs.Root
-                  style={{margin: '12px 16px 0 16px'}}
-                  value={tabs[tabValue].label}
-                  onValueChange={handleTabChange}>
-                  <Tabs.List>
-                    {tabs.map(tab => (
-                      <Tabs.Trigger
-                        key={tab.label}
-                        value={tab.label}
-                        className="h-[30px] text-sm">
-                        {tab.label}
-                      </Tabs.Trigger>
-                    ))}
-                  </Tabs.List>
-                </Tabs.Root>
-              )}
-              <Box
-                sx={{
-                  overflow: 'hidden',
-                  display: 'flex',
-                  flexDirection: 'column',
-                  flex: '1 1 auto',
-                }}>
-                <ErrorBoundary key={tabId}>{tabContent}</ErrorBoundary>
-              </Box>
-            </Box>
+            <SplitPanelRight
+              minWidth={150}
+              defaultWidth={200}
+              maxWidth="50%"
+              drawer={props.rightSidebarContent}
+              isDrawerOpen={props.isRightSidebarOpen ?? false}
+              main={
+                <MainPeekContent
+                  headerContent={props.headerContent}
+                  tabContent={tabContent}
+                  tabs={props.tabs}
+                  tabId={tabId}
+                  tabValue={tabValue}
+                  hideTabsIfSingle={props.hideTabsIfSingle}
+                  handleTabChange={handleTabChange}
+                />
+              }
+            />
           }
         />
       </div>
     </Box>
   );
 };
 
+const MainPeekContent: FC<{
```

Comment:
nit: This view is not just for peeking. It is visible when expanded. Suggest renaming to something like `SimpleTabView` to more accurately reflect the purpose